src/core/electrostatics_magnetostatics/coulomb.cpp

```diff
@@ -42,8 +42,7 @@
 Coulomb_parameters coulomb;
 
 namespace Coulomb {
-void calc_pressure_long_range(Observable_stat &virials,
-                              Observable_stat &p_tensor,
+void calc_pressure_long_range(Observable_stat &p_tensor,
```

Comment:
Also here I think you can just return the value (or multiple vectors, if you want to keep the elc accounting, I have dropped that).

Reply:
It would be convenient if we could also drop the separate counting of the dipolar correction slot. @RudolfWeeber what do you think about adding the DLC energy correction from `add_mdlc_energy_corrections()` directly into the slot where the DP3M/DS energy is stored? This mechanism for storing ELC and DLC corrections in a separate slot of `Observable_stat` looks like a debug feature to me.

Reply:
I agree, but you can also just return an vector for now and we sort this out later.

Reply:
Well that would be a `Vector2d` because we cannot overwrite the first slot, which comes from particles. This means adding a for loop with an offset or manually assigning the two values. If we remove the ELC slot, it makes sense to remove the DLC too, to make the API change consistent. IIRC Rudolf only needed two slots.

Reply:
I'd return an array with 3 values, because there are three slots. Otherwise the concerns are not separated because the caller has to know which slot is which/which one is set

Reply:
This means we need a `for` loop to add the `Vector3d` values instead of doing a copy.

Reply:
`boost::copy(calc_pressure_long_range(...), obs_energy.coulomb)` ?

Reply:
wouldn't that overwrite the first slot of `obs_energy.coulomb`, which comes from the short-range loop?

Reply:
Ah I see what you mean... Would be easier if `Observvable_stat` would have a proper value type :D